### PR TITLE
Add extra "sanitize"

### PR DESCRIPTION
**Pull request by plenaerts**
Cleaning up my requirements I noted that I had installed nh3 as a dependency, while it was an optional for django-prose-editor.

I suggest this extra "sanitize" to keep my project dependencies cleaner.

### CODE REVIEW

**Comment by matthiask** [comment]
Awesome, thank you! Do you find the time to add this to the README as well?

**Comment by plenaerts** [comment]
I added a proposal for the README.

File: README.rst [hunk 1]
```diff
@@ -65,6 +72,8 @@ snippet to always pass HTML through `nh3
 
     description = SanitizedProseEditorField()
 
+Install django-prose-editor with the extra "sanitized" to use
+SanitizedProseEditorField.
```
Comment (matthiask) [inline README.rst:76]:
```suggestion
Install django-prose-editor with the extra "sanitize" to use
``SanitizedProseEditorField``.
```

**Comment by matthiask** [comment]
Released as 0.6.1. Thank you!

**Pull request merged**